bundle/edu.gemini.lchquery.servlet/src/main/scala/edu/gemini/lchquery/servlet/LchQueryFunctor.scala

```diff
@@ -77,6 +77,12 @@ class LchQueryFunctor(queryType: LchQueryFunctor.QueryType,
         Some(new NonSidereal() {
           setName(target.getName)
           setType(targetType.orNull)
+          for {
+            n <- target.getNonSiderealTarget
+            h <- n.horizonsDesignation
+          } {
+            setHorizonsObjectId(h.des)
+          }
```

Comment:
Huh. Yes, I see this now with Mars Barycenter and Vesta, which both seem to have an integer of 4 assigned to them. Thanks for pointing this out. I'll pass in the `queryString` instead of the `des`.